### PR TITLE
Enable block and state level reports for BD

### PR DESCRIPTION
**Pull request by claudiovallejo**
**Story card:** [ch4535](https://app.clubhouse.io/simpledotorg/story/4535/enable-state-and-block-level-reports-for-bangladesh)

## Because

We need to enable block and state-level reports for Bangladesh as we add 1,300+ new facilities.

## This addresses

Make `extended_region_reports` equal `true`

## Test instructions

Check that the `extended_region_reports` key is `true`

### CODE REVIEW

**Review by harimohanraj89 — APPROVED** [review]
:shipit:

**Comment by rsanheim** [comment]
You'll need to update the failing tests for this.

Also, it would be good to change your local env to BD and just click around to test this out - it should be fine but doesn't hurt.

File: spec/models/region_spec.rb [hunk 1]
```diff
@@ -61,24 +61,17 @@
...
     end
 
-    it "excludes states and blocks for other countries" do
-      allow(CountryConfig).to receive(:current).and_return(CountryConfig.for(:BD)).at_least(:once)
+    it "is everything for Bangladesh" do
+      expect(CountryConfig).to receive(:current).and_return(CountryConfig.for(:BD)).at_least(:once)
 
       org = Seed.seed_org.region
-      state = FactoryBot.create(:region, :state, reparent_to: org)
+      state = FactoryBot.create(:region, :state, reparent_to: Seed.seed_org.region)
```
Comment (rsanheim) [inline spec/models/region_spec.rb:68]: You can use the local org here, just a tiny bit more clear.

```suggestion
      state = FactoryBot.create(:region, :state, reparent_to: org)
```

**Review by rsanheim — APPROVED** [review]
LGTM, one small suggestion

**Pull request merged**